content/blog/2024/value-proposition/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: An update to 2i2c's value proposition
+date: "2024-05-06"
+lastmod: "2024-05-06"
+authors: ["Chris Holdgraf"]
+tags: [product]
+categories: [updates]
+featured: false
+draft: false
+---
+
+2i2c recently finished [it's three year retrospective](https://2i2c.org/report-czi-2021) after closing out its original seed grant.
+As part of this process, we're revisiting our organizational strategy, structure, and overall plan, in order to incorporate the learning that we've done over our first three years.
+
+Our **value proposition** is a key framing for 2i2c's intended impact, key stakeholders, and platform strategy.
+We recently completed a team exercise to refine this statement for 2i2c, and are excited to share it with you below:
+
+<style>
+  .pull-quote {
+    padding-left: 1rem;
+    border-left: .25rem solid #1d4ef5;
+    margin-bottom: 1rem;
+  }
+  .pull-title {
+    font-size: 1.5rem;
+    margin-bottom: .5rem;
+    font-weight: bold;
+  }
+  .pull-subtitle {
+    font-size: 1rem;
+    font-style: italic;
+  }
+</style>
+<div class="pull-quote">
+  <p class="pull-title">
+  A global network of community hubs for interactive learning and discovery
+  </p>
+  <p class="pull-subtitle">
+  Our interactive computing platform gives your community a digital home to create and share knowledge, and a global network of communities to learn from.
+  </p>
+</div>
+
+We believe that this leans into and builds upon the way we've described our service so far.
+It recognizes that the value in our service doesn't end with simple access to an interactive computing environment, but lies in how that environment's tools empower a community to create and share knowledge with one another.
+
+We believe that this is an exciting direction to build towards, because it is based in the value of the collection of communities we serve, and the way that this network can be greater than the sum of its parts.
```

Comment:
The use of "We believe" twice in quick proximity hits a little odd to me, I would just start this sentence with "This is an exciting direction..."